### PR TITLE
TEN Validator Cloud Deployment Guide

### PR DESCRIPTION
**Pull request by anixon604**
Node Operator documentation in sync with https://github.com/ten-protocol/ten-validator
- [x] finalize architecture section
- [x] review

### CODE REVIEW

File: docs/testnet/for-validators/overview.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## Transaction Flow (TX)
+![Ten Validator Flow](../../assets/ten-validator-flow.png)
+
+### 1. Transactions
+Transactions are the fundamental operations executed on the Ten network, representing actions such as transferring value, interacting with smart contracts, or executing other network functions. Each transaction serves as a discrete unit of activity, altering the state of the network based on its payload.
+
+### 2. Batches
+Batches consist of multiple transactions grouped together on the Ten network, functioning similarly to blocks on the Ethereum network. By aggregating transactions into batches, the network enhances throughput and lowers latency, optimizing resource usage and improving overall efficiency.
```
Comment (GavT) [inline docs/testnet/for-validators/overview.md:15]: TEN not Ten

File: docs/testnet/for-validators/overview.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### 3. Roll-ups
+Roll-ups are a layer 2 scaling mechanism employed by the Ten network to securely aggregate and process transactions off-chain, before submitting a compressed version of these transactions to the Ethereum mainnet (Layer 1) for finalization. This approach significantly boosts throughput and reduces latency while maintaining the security guarantees of the Ethereum network.
```
Comment (GavT) [inline docs/testnet/for-validators/overview.md:18]: TEN not Ten

File: docs/testnet/for-validators/node-architecture.md
Comment (anixon604) [inline docs/testnet/for-validators/node-architecture.md:1]: The Node Composition section is missing a diagram, will add it in after. Just a simple zoom on the Host/Enclave/EdgelessDB compositions. @GavT

File: docs/testnet/for-validators/overview.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Node Types
+There are two types of nodes in the Ten network: Sequencers and Validators. They use the same software but have different roles and responsibilities. As a node operator you will be running a Validator node, but it is important to understand the role of Sequencers in the network.
+
+### Sequencer
+The TEN Sequencer is a central node that is responsible for ordering transactions, generating new batches, and creating roll-ups on the Ten network. It is the only node that can create new batches and is responsible for broadcasting these to the network, as well as submitting them to the Ethereum mainnet via roll-up.
+
+
+### Validator
+A Ten Validator is decentralized node that participates in the Ten network by processing transactions, validating transactions and batches against Ethereum L1 roll-ups, and providing data availability. The participation of multiple validators ensures the network’s security and integrity.
```
Comment (GavT) [inline docs/testnet/for-validators/overview.md:31]: TEN not Ten.
Worth expanding on the validation slightly? i.e. checking the rollup is intact / not been meddled with. This means validators also ensure the network is trustworthy.

File: docs/testnet/for-validators/rewards.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+---
+# Node Rewards
+
+As a node operator, your participation is essential in maintaining a secure, reliable, and high-performance network. In recognition of your contributions, TEN has implemented a comprehensive reward structure to incentivize active and consistent engagement in the network’s testnet phases.
+
+---
+
+## Program Overview
+
+TEN is an encrypted Layer 2 network built on Ethereum, designed by the team behind Corda. TEN offers a highly scalable and private environment for Web3 applications,including Ai, tokenizing real-world assets (RWA), enabling decentralized financial solutions (DeFi) and on-chain games.
```
Comment (GavT) [inline docs/testnet/for-validators/rewards.md:12]: AI rather than Ai

File: docs/testnet/for-validators/rewards.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+---
+
+## Node Operator Rewards Structure
+
+TEN has allocated **22,500,000 TEN** (representing 2.25% of the total token supply) to incentivize contributions across the community, including node operations.
+
+### Rewards Overview
+
+1. **Staking Requirements**: To participate as a node operator in the testnet requires no initial stake, (however a minimum of **50,000 TEN tokens** must be staked will be required on Mainnet. This ensures a commitment to maintaining network performance.)
```
Comment (GavT) [inline docs/testnet/for-validators/rewards.md:26]: grammar on the mainnet staking is mangled

File: docs/testnet/for-validators/rewards.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+### Performance-Based Rewards
+
+Performance-based rewards are structured to motivate the consistent and high-quality performance of nodes, as follows:
+
+- **Top 10% Performers**: The highest 10% of node operators, ranked by performance and uptime, will earn the top reward bracket. Top performers will receive a reward equivalent to the minimum mainnet staking requirement and will be offered priority access to participate in the mainnet launch.
+- **Remaining Participants**: Node operators outside the top 10% but in the top 20% will also receive a reward based on their contributions and performance.
+
+### Participation-Based Rewards
+
+Node operators who meet minimum participation requirements but do not rank in the top 20% will still receive a token reward. These rewards aim to foster inclusivity and recognize the contributions of all node operators.
```
Comment (GavT) [inline docs/testnet/for-validators/rewards.md:39]: "receive a reward" not "receive a token reward" ie remove "token"

File: docs/testnet/for-validators/rewards.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+---
+
+## Infrastructure Stipend
```
Comment (GavT) [inline docs/testnet/for-validators/rewards.md:43]: I'm not aware of this stipend. Suggest removing it for now

File: docs/testnet/for-validators/rewards.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+The incentivized testnet is split into three phases, each lasting approximately **6-8 weeks**, followed by a short intermission between phases. The phases are designed to test and validate different aspects of the network:
+
+1. **Phase 1**: Focus on onboarding developers and user quests aimed at network resilience.
+2. **Phase 2**: Dedicated to expanding node operations and enhancing network decentralization.
+3. **Phase 3**: Final phase with an emphasis on testing newly deployed dApps and security-focused initiatives.
+
+### Eligibility
+
+- **Application Deadline**: Node operators must apply by **March 1, 2024**.
```
Comment (GavT) [inline docs/testnet/for-validators/rewards.md:59]: Remove the deadline date and just make it "Application: node operators must apply". Is there a form to use?

File: docs/testnet/for-validators/rewards.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+- **Selection**: Applications are prioritized based on node management experience and community commitment, with selected participants notified by email.
+- **Jurisdictional Limitations**: Participation is restricted to users in non-U.S./Canada jurisdictions, compliant with OFAC and applicable local laws.
```
Comment (GavT) [inline docs/testnet/for-validators/rewards.md:61]: I'd soften this and just say "jurisdictional limitations" will be applied

File: docs/testnet/for-validators/rewards.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+## Additional Information
+
+- **Updates and Notifications**: Important updates will be shared via the **Ten Discord testnet channel** and by email for selected participants.
```
Comment (GavT) [inline docs/testnet/for-validators/rewards.md:65]: TEN not Ten

Comment (GavT) [inline docs/testnet/for-validators/rewards.md:65]: Link to the Discord channel to be added

File: docs/testnet/for-validators/rewards.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+All rewards will be distributed after the mainnet launch, following the tokenomics vesting schedule outlined in the official TEN whitepaper.
+
+---
+
+This Node Rewards program aims to build a community-driven network powered by participants' shared efforts and contributions. By rewarding commitment and performance, TEN is cultivating a stable, high-performance Layer 2 ecosystem that is decentralized, secure, and inclusive.
+
+---
+
+For further queries, please reach out via our **[Discord channel](#)** or visit our **[official documentation](#)**.
```
Comment (GavT) [inline docs/testnet/for-validators/rewards.md:78]: Links need adding

File: docs/testnet/for-validators/running-a-node.md [hunk 12]
```diff
@@ -1,12 +1,93 @@
 ---
-sidebar_position: 3
+sidebar_position: 4
 ---
-# Running a node
+# Running a TEN Validator Node on Azure
 
-Thank you for your interest in running a node on the Ten testnet.
+This guide provides detailed instructions to set up and manage a TEN Validator Node on Azure using Terraform and Ansible. Due to TEN’s reliance on SGX architecture and cloud-specific security assurances, all current deployments are limited to cloud infrastructure to mitigate risks associated with physical attacks. A proof-of-cloud assertion, provided by TEN, certifies the security and reliability of each validator node running in the cloud.
```
Comment (GavT) [inline docs/testnet/for-validators/running-a-node.md:6]: "use of" rather than "reliance"

**Review by 0xjba — APPROVED** [review]
LGTM! Neat & the Azure part looks super simple.

**Pull request merged**